BaseButton repr causes AttributeError

The `__repr__` implementation of `BaseButton` references `self.args` and `self.kwargs`, which are only set in WidgetDescriptor.

This causes an AttributeError when `__repr__` is called on an `ouia.Button` widget, which is easily reproducible in the unit tests:

``` git diff
diff --git a/testing/ouia/test_button.py b/testing/ouia/test_button.py
index 1c5b8f8..77b3f47 100644
--- a/testing/ouia/test_button.py
+++ b/testing/ouia/test_button.py
@@ -16,5 +16,6 @@ def view(browser):
 
 
 def test_button_click(view):
+    view.button.__repr__()
     assert view.button.is_displayed
     view.button.click()
❯ BROWSER=firefox pytest -v testing/ouia/test_button.py
============================================================================================================ test session starts ============================================================================================================
platform linux -- Python 3.8.11, pytest-6.2.4, py-1.10.0, pluggy-0.13.1 -- /home/mshriver/repos/redhatqe/widgetastic.patternfly4/.venv/bin/python3.8
cachedir: .pytest_cache
rootdir: /home/mshriver/repos/redhatqe/widgetastic.patternfly4, configfile: pytest.ini
plugins: forked-1.3.0, xdist-2.3.0, cov-2.12.1
collected 1 item                                                                                                                                                                                                                            

testing/ouia/test_button.py::test_button_click FAILED                                                                                                                                                                                 [100%]

================================================================================================================= FAILURES ==================================================================================================================
_____________________________________________________________________________________________________________ test_button_click _____________________________________________________________________________________________________________

view = <test_button.view.<locals>.TestView object at 0x7f3ee8ac0dc0>

    def test_button_click(view):
>       view.button.__repr__()

testing/ouia/test_button.py:19: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <[AttributeError("'Button' object has no attribute 'args'") raised in repr()] Button object at 0x7f3ee8ac0c70>

    def __repr__(self):
>       return "{}{}".format(type(self).__name__, call_sig(self.args, self.kwargs))
E       AttributeError: 'Button' object has no attribute 'args'
```

--- a/src/widgetastic_patternfly4/button.py
+++ b/src/widgetastic_patternfly4/button.py
@@ -1,4 +1,3 @@
-from widgetastic.log import call_sig
 from widgetastic.utils import ParametrizedLocator
 from widgetastic.widget import ClickableMixin
 from widgetastic.widget import Widget
@@ -37,7 +36,7 @@ class BaseButton:
         return check1 or check2 or self.browser.get_attribute("disabled", self) is not None
 
     def __repr__(self):
-        return "{}{}".format(type(self).__name__, call_sig(self.args, self.kwargs))
+        return "{}{}".format(type(self).__name__, self.locator)
 
     @property
     def title(self):

--- a/testing/ouia/test_button.py
+++ b/testing/ouia/test_button.py
@@ -16,5 +16,6 @@ def view(browser):
 
 
 def test_button_click(view):
+    view.button.__repr__()
     assert view.button.is_displayed
     view.button.click()

